docs/guides/redis/monitoring/using-builtin-prometheus.md

```diff
@@ -58,318 +61,258 @@ spec:
-redis-mon-prometheus-stats   ClusterIP   10.104.85.239   <none>        56790/TCP   1m
+$ kubectl describe svc -n demo builtin-prom-redis-stats
+Name:              builtin-prom-redis-stats
+Namespace:         demo
+Labels:            kubedb.com/kind=Redis
+                   kubedb.com/name=builtin-prom-redis
+Annotations:       monitoring.appscode.com/agent: prometheus.io/builtin
+                   prometheus.io/path: /metrics
+                   prometheus.io/port: 56790
+                   prometheus.io/scrape: true
+Selector:          kubedb.com/kind=Redis,kubedb.com/name=builtin-prom-redis
+Type:              ClusterIP
+IP:                10.106.243.251
+Port:              prom-http  56790/TCP
+TargetPort:        prom-http/TCP
+Endpoints:         172.17.0.5:56790
+Session Affinity:  None
+Events:            <none>
 ```
 
-```yaml
-$ kubectl get services redis-mon-prometheus-stats -n demo -o yaml
-apiVersion: v1
-kind: Service
-metadata:
-  annotations:
-    monitoring.appscode.com/agent: prometheus.io/builtin
-    prometheus.io/path: /metrics
-    prometheus.io/port: "56790"
-    prometheus.io/scrape: "true"
-  creationTimestamp: 2018-10-01T06:35:03Z
-  labels:
-    kubedb.com/kind: Redis
-    kubedb.com/name: redis-mon-prometheus
-  name: redis-mon-prometheus-stats
-  namespace: demo
-  ownerReferences:
-  - apiVersion: kubedb.com/v1alpha1
-    blockOwnerDeletion: false
-    kind: Redis
-    name: redis-mon-prometheus
-    uid: 076b13b3-c544-11e8-9ba7-0800274bef12
-  resourceVersion: "10495"
-  selfLink: /api/v1/namespaces/demo/services/redis-mon-prometheus-stats
-  uid: 211f339b-c544-11e8-9ba7-0800274bef12
-spec:
-  clusterIP: 10.104.85.239
-  ports:
-  - name: prom-http
-    port: 56790
-    protocol: TCP
-    targetPort: prom-http
-  selector:
-    kubedb.com/kind: Redis
-    kubedb.com/name: redis-mon-prometheus
-  sessionAffinity: None
-  type: ClusterIP
-status:
-  loadBalancer: {}
+You can see that the service contains following annotations.
+
+```console
+prometheus.io/path: /metrics
+prometheus.io/port: 56790
+prometheus.io/scrape: true
 ```
 
-We can see that the service contains these specific annotations. The Prometheus server will discover the exporter using these specifications.
+The Prometheus server will discover the service endpoint using these specifications and will scrape metrics from exporter.
+
+## Configure Prometheus Server
+
+Now, we have to configure a Prometheus scrapping job to scrape the metrics using this service. We are going to configure scrapping job similar to this [kubernetes-service-endpoints](https://github.com/appscode/third-party-tools/tree/master/monitoring/prometheus/builtin#kubernetes-service-endpoints) job that scrapes metrics from endpoints of a service.
+
+Let's configure a Prometheus scrapping job to collect metrics from this service.
 
 ```yaml
-prometheus.io/path: ...
-prometheus.io/port: ...
-prometheus.io/scrape: ...
+- job_name: 'kubernetes-service-endpoints'
```

Comment:
How are we targeting this specific service?

Reply:
We are not. It is generic configuration. User may have already have this configuration to monitor other resources. So, they don't need to update anything.